tools/oas-to-am/README.md

```diff
@@ -0,0 +1,30 @@
+# OpenAPI Specification to Assign Message
+
+A simple tool to generate an Assign Message policy from a JSON OpenAPI Spec.
+
+## Usage
+
+``` sh
+git clone https://github.com/apigee/devrel
+cd devrel/tools/oas-to-am
+export PATH=$PATH:$PWD/bin #you can add this to your .bashrc too
+SPEC=./path/to/spec.json OPERATION=operationId oas-to-am
```

Comment:
I think OPERATION needs to be explained. Alternatively skip it and just generate all operations?

Reply:
How about reading the spec from an argument instead? I don't necessarily care but `oas-to-asm ./path/to/spec.json` looks slightly easier to me.

Reply:
for now, i will leave out argument parsing to keep this project small. the current design is to be used by the openlegacy cli and not be consumed directly, however other PRs can consider this